addons/fleet/views/fleet_vehicle_model_views.xml

```diff
@@ -72,23 +72,12 @@
                         <page string="Vendors" name="vendors">
                             <field name="vendors">
                                 <kanban quick_create="false" create="true">
-                                    <field name="name"/>
-                                    <field name="phone"/>
-                                    <field name="email"/>
                                     <templates>
-                                        <t t-name="kanban-box">
-                                            <div style="position: relative" class="oe_kanban_global_click">
-                                                <div>
-                                                    <div class="o_kanban_record_title">
-                                                        <field name="name"/>
-                                                        <div class="o_kanban_details float-end">
-                                                            <span class="text-muted">
-                                                                <t t-if="record.phone.raw_value"><field name="phone"/><br/></t>
-                                                                <t t-if="record.email.raw_value"><field name="email"/></t>
-                                                            </span>
-                                                        </div>
-                                                    </div>
-                                                </div>
+                                        <t t-name="kanban-card">
+                                            <div>
```

Comment:
div not necessary

Reply:
so for that i have used flex-row and put phone and email field in one div.